src/me/fru1t/subedit/utils/Utils.kt

```diff
@@ -7,52 +7,120 @@ import java.io.File
  * multiple tools.
  */
 object Utils {
-  private const val NEWLINE = "\r\n"
+  public const val SUB_NEWLINE = "\\N"
 
-  /** Prompts the user for a file. Returns the file if valid or `null` if no file could be prodded from the user. */
-  fun askForFile(): File? {
+  private val DIALOGUE_REGEX = Regex("^Dialogue: ([^,]+),([^,]+),([^,]+),([^,]+),([^,]*),(\\d+),(\\d+),(\\d+),([^,]*),(.*)\$")
+  private val COLOR_REGEX = Regex("[0-9A-F]{6}")
+  private val ALPHA_REGEX = Regex("[0-9A-F]{2}")
```

Comment:
Does .ass accept both upper and lowercase? Should we guard against that or force a specific format?

Reply:
Agreed to accept only uppercase